### ISSUE TITLE
[DOC] write docstring for RNN estimators

### ISSUE THREAD

**Issue by fkiraly** [sktime/sktime#4558]
The estimators `SimpleRNNRegressor` and `SimpleRNNClassifier` are practically empty, these should be added.

**Comment by wasup-yash** [comment]
hey, I like to work on this issue and as I am new to the project could you provide the references to solve the issue?

**Comment by fkiraly** [comment]
sure!

have a look at other estimators, e.g., `KNeighborsTimeSeriesClassifier`, and write the docstring in the same format.

If you are wondering about how to contribute, here is the contribution guide:
https://www.sktime.net/en/latest/get_involved/contributing.html

**Comment by wasup-yash** [comment]
Thanks for the instructions @fkiraly

**Comment by fkiraly** [comment]
welcome to sktime, @wasup-yash!

**Comment by achieveordie** [comment]
Yikes, must have missed adding that. @wasup-yash thanks for taking it up, if you have any questions regarding it please let us know.

**Comment by wasup-yash** [comment]
@achieveordie yeah for sure can you please point out where exactly in path it needs to be updated ?

**Comment by achieveordie** [comment]
Take a look at [`CNNClassifier`](https://github.com/sktime/sktime/blob/main/sktime/classification/deep_learning/cnn.py#L17) for comparison. It mentions all the parameters that are passed to the constructor. If you look at [`SimpleRNNClassifier`](https://github.com/sktime/sktime/blob/main/sktime/classification/deep_learning/rnn.py#L20), it only has a reference and provides no information about the parameters. This is also true for [`SimpleRNNRegressor`](https://github.com/sktime/sktime/blob/main/sktime/regression/deep_learning/rnn.py#L20).

We use [Read The Docs](https://github.com/sktime/sktime/blob/main/.readthedocs.yml) and [Sphinx](https://github.com/sktime/sktime/blob/main/docs/source/conf.py) to auto-generate the documentation using the docstrings.

Also, before you start working on it, please ensure that you've got the development setup right- this will ensure the linting/formatting checks won't fail when you open a PR. Here's the [developer's guide](https://www.sktime.net/en/stable/developer_guide.html) to assist you with it.

**Comment by wasup-yash** [comment]
> 

hey the parameters are the same in the `[SimpleRNNClassifier](https://github.com/sktime/sktime/blob/main/sktime/classification/deep_learning/rnn.py#L20)` and `[SimpleRNNRegressor](https://github.com/sktime/sktime/blob/main/sktime/regression/deep_learning/rnn.py#L20)` right ? and we need to only add parameters only into it?

**Comment by achieveordie** [comment]
There you go @wasup-yash, thanks :)

**Comment by wasup-yash** [comment]
thanks, @achieveordie , I'll reach out incase of some issue :)

**Comment by wasup-yash** [comment]
hey @achieveordie do `SimpleRNNClassifier` and `SimpleRNNRegressor` have any like references to add ?like in  `CNNClassifier` we have `Notes
    -----
    Adapted from the implementation from Fawaz et. al
    https://github.com/hfawaz/dl-4-tsc/blob/master/classifiers/cnn.py`

**Comment by achieveordie** [comment]
We're already adding the references, aren't we? [Line 24](https://github.com/sktime/sktime/blob/main/sktime/regression/deep_learning/rnn.py#L24)

**Comment by wasup-yash** [comment]
> We're already adding the references, aren't we? [Line 24](https://github.com/sktime/sktime/blob/main/sktime/regression/deep_learning/rnn.py#L24)

yeah but in `CNNClassifier` there was an extra ref as a note so that's why I had a doubt, thanks for helping
